88.Merge_Sorted_Array.py

```diff
@@ -0,0 +1,30 @@
+# https://leetcode.com/problems/merge-sorted-array/
+
+def merge(nums1, m, nums2, n):
+    i = m - 1
+    j = n - 1
+    l = m + n - 1
+
+    while i >= 0 and j >= 0:
```

Comment:
you can do l -= 1 only once in the end if if-else, anyway you do it for both cases